.github/workflows/publish.yml

```
@@ -0,0 +1,34 @@
+name: Publish Release
+
+on:
+  workflow_dispatch:
+  release:
+    types:
+      - published
+
+permissions:
+  contents: read
+
+jobs:
+  publish-npm:
+    name: 'NPM'
```

This should probably be a more human readable string.
```suggestion
    name: Publish to NPM
```